### PR TITLE
docs: Update Github workflow status shields

### PR DESCRIPTION
**Pull request by roniemartinez**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### Other information

There was a breaking change for this badge endpoint in https://github.com/badges/shields/issues/8671

image

FIxed badge:

image

Also note that `test-library` was changed to `test-core` but not sure whether I also change it.

### CODE REVIEW

**Review by FabianLars — DISMISSED** [review]
Thank you!

**Comment by FabianLars** [comment]
Actually, on second thought, how about we replace it with github's own badge feature https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge ? I really like the github icon it has then tbh 😅

**Comment by roniemartinez** [comment]
@FabianLars we can also add github's logo into shields.io

```
https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20core&logo=github
```

image


If it's from Github's own badge, it should look like this (**shows darker icon**):

image


Let me know which one you prefer.

**Comment by FabianLars** [comment]
ah yeah, that one works too. Let's go with that then 👍 ~~And while we're at it maybe rename the label to just `CI` for now? What do you think?~~ nvm too many different workflows

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![status](https://img.shields.io/badge/status-stable-blue.svg)](https://github.com/tauri-apps/tauri/tree/dev)
 [![License](https://img.shields.io/badge/License-MIT%20or%20Apache%202-green.svg)](https://opencollective.com/tauri)
-[![test library](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library)](https://github.com/tauri-apps/tauri/actions?query=workflow%3A%22test+library%22)
+[![test library](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20library)](https://github.com/tauri-apps/tauri/actions/workflows/test-core.yml)
```
Comment (amrbashir) [inline README.md]:
```suggestion
[![test core](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20core)](https://github.com/tauri-apps/tauri/actions/workflows/test-core.yml)
```

and same for all other READMEs

File: core/tauri-codegen/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 [![Chat Server](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/SpmNs4S)
 [![devto](https://img.shields.io/badge/blog-dev.to-black.svg)](https://dev.to/tauri)
 
-![](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library)
+![](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20library)
```
Comment (amrbashir) [inline core/tauri-codegen/README.md]:
```suggestion
[![test core](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20core)](https://github.com/tauri-apps/tauri/actions/workflows/test-core.yml)
```
and repeat for others that don't have links

File: tooling/cli/node/README.md [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 [![status](https://img.shields.io/badge/status-stable-blue.svg)](https://github.com/tauri-apps/tauri/tree/dev)
 [![License](https://img.shields.io/badge/License-MIT%20or%20Apache%202-green.svg)](https://opencollective.com/tauri)
-[![test library](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library)](https://github.com/tauri-apps/tauri/actions?query=workflow%3A%22test+library%22)
+[![test library](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20library)](https://github.com/tauri-apps/tauri/actions/workflows/test-core.yml)
```
Comment (amrbashir) [inline tooling/cli/node/README.md]:
```suggestion
[![test cli](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-cli-js.yml?label=test%20cli)](https://github.com/tauri-apps/tauri/actions/workflows/test-cli-js.yml)
```

File: tooling/cli/README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@
 
 [![status](https://img.shields.io/badge/status-stable-blue.svg)](https://github.com/tauri-apps/tauri/tree/dev)
 [![License](https://img.shields.io/badge/License-MIT%20or%20Apache%202-green.svg)](https://opencollective.com/tauri)
-[![test library](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library)](https://github.com/tauri-apps/tauri/actions?query=workflow%3A%22test+library%22)
+[![test library](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20library)](https://github.com/tauri-apps/tauri/actions/workflows/test-core.yml)
```
Comment (amrbashir) [inline tooling/cli/README.md]:
```suggestion
[![test cli](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-cli-rs.yml?label=test%20cli)](https://github.com/tauri-apps/tauri/actions/workflows/test-cli-rs.yml)
```

File: tooling/api/README.md [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 [![status](https://img.shields.io/badge/status-stable-blue.svg)](https://github.com/tauri-apps/tauri/tree/dev)
 [![License](https://img.shields.io/badge/License-MIT%20or%20Apache%202-green.svg)](https://opencollective.com/tauri)
-[![test library](https://img.shields.io/github/workflow/status/tauri-apps/tauri/test%20library?label=test%20library)](https://github.com/tauri-apps/tauri/actions?query=workflow%3A%22test+library%22)
+[![test library](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/test-core.yml?label=test%20library)](https://github.com/tauri-apps/tauri/actions/workflows/test-core.yml)
```
Comment (amrbashir) [inline tooling/api/README.md]:
```suggestion
[![lint js](https://img.shields.io/github/actions/workflow/status/tauri-apps/tauri/lint-js.yml?label=lint%20js)](https://github.com/tauri-apps/tauri/actions/workflows/lint-js.yml)
```

File: README.md (hunk 1 above)
Comment (roniemartinez) [reply README.md]: This was already on the description above. Then I will change it to core.

> Also note that test-library was changed to test-core but not sure whether I also change it.

Comment (amrbashir) [reply README.md]: my bad, I didn't notice it, thanks for your contribution.

**Comment by roniemartinez** [comment]
@FabianLars @amrbashir I updated in 885493d3bc9d97ac2a2e17769e71b4651a6181f3

**Review by amrbashir — APPROVED** [review]

**Pull request merged**